src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -0,0 +1,33 @@
+import { FC } from 'react';
+import classNames from 'classnames';
+import { FilterTodosBy } from '../../types/FilterTodosBy';
+
+type Props = {
+  setFilterBy: (FilterTodosBy: FilterTodosBy) => void,
+  filterBy: string,
```

Comment:
The type of 'filterBy' should be 'FilterTodosBy' instead of 'string' to match the expected type.